code/model/ForumThread.php

```diff
@@ -133,7 +133,7 @@ function getFirstPost() {
 	 * @return int
 	 */
 	function getNumPosts() {
-		return (int)DB::query("SELECT count(*) FROM \"Post\" WHERE \"ThreadID\" = $this->ID")->value();
+		return (int)DB::query(sprintf('SELECT COUNT(Post.ID) FROM Post LEFT JOIN Member ON Post.AuthorID = Member.ID WHERE ThreadID = %s AND Member.ForumStatus = \'Normal\'', $this->ID))->value();
```

Comment:
Please keep line length below 120 chars.

Can't we use the ORM to build this?

Something like:

```php
return Post::get()->filter(array(
    'Author.ForumStatus ' => 'Normal',
))->count();
```